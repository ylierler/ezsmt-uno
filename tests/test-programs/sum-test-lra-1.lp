&logic(lra).
&sum{x; "-1.8"*y;7*z}="7.8".