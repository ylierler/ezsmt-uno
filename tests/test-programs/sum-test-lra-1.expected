{}
z=0.0 x=(/ 39 5) y=0.0 